src/Client/pages/WishList.fs

```diff
@@ -115,19 +114,19 @@ let update (msg:Msg) model : Model * Cmd<Msg> =
             | Some err ->
                 { model with ErrorMsg = Some err }, Cmd.none
             | None ->
-                let wishList = 
-                    { model.WishList 
-                        with 
-                            Books = 
-                                (model.NewBookModel.NewBook :: model.WishList.Books) 
+                let wishList =
+                    { model.WishList
+                        with
+                            Books =
+                                (model.NewBookModel.NewBook :: model.WishList.Books)
                                 |> List.sortBy (fun b -> b.Title) }
 
                 let submodel,cmd = NewBook.init()
                 { model with WishList = wishList; NewBookModel = submodel; ErrorMsg = None },
                     Cmd.batch [
                         Cmd.map NewBookMsg cmd
-                        Cmd.ofPromise postWishList (model.Token,wishList) FetchedWishList FetchError
-                    ]            
+                        Cmd.OfPromise.either postWishList (model.Token,wishList) FetchedWishList FetchError
```

Comment:
urgs!? what's that?

Reply:
A long discussion lead to this changes: https://github.com/elmish/elmish/issues/166